al_log_filter.js

```diff
@@ -7,16 +7,16 @@
  * @end
  * -----------------------------------------------------------------------------
  */
-const where = require('lodash.where');
+const filterItem = require('lodash.filter');
```

Comment:
let's rename it to `lodashFilter` or similar.

Reply:
Done